R/model-summary.R

```diff
@@ -134,12 +134,14 @@
 #' See [print_bbi_args()] for full list of options.
 #' @param ... args passed through to [bbi_exec()]
 #' @param .dry_run show what the command would be without actually running it
+#' @param .overwrite whether or not to overwrite values of error/missing values (defaults to -1e+09) with `NA`
```

Comment:
I suggest writing out the nines to avoid any confusion about whether the value is `-999999999` or `-1000000000`

```
> -999999999
[1] -1e+09
> -999999999L
[1] -999999999
> -1e9
[1] -1e+09
> -1e9L
[1] -1000000000
> identical(-1000000000, -1e9)
[1] TRUE
> identical(-1000000000L, -1e9L)
[1] TRUE
```

Reply:
Interesting, didnt realize that. I reverted back a commit, so that new argument wont be there, but sounds good in general. Will look for cases of that

Reply:
🤯